### ISSUE TITLE
<amp-mathml> only supports block mode

### ISSUE THREAD

**Issue by KENNYSOFT** [ampproject/amphtml#19420]
## What's the issue?

`<amp-mathml>` only supports block mode, which is in `\[ ... \]` or `$$ ... $$` form. ASCIImath with \` delimiters only produces non-block mode `div` and `\( ... \)` form also, so it needs to be changed with handling `inline` attribute. (The difference with `\[ ... \]` and `\( ... \)` is not only the block mode but also the display style especially for fractions and etc.) Suggested changes are followed.

## How do we reproduce the issue?

Normal HTML with next source (using normal MathJax) generates the next figure.
```html
<p>\[x = {-b \pm \sqrt{b^2-4ac} \over 2a}\]</p>
<p>MathJax \(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\) MathML \( \cos(θ+φ) \) AMP</p>
```
![image](https://user-images.githubusercontent.com/6624567/48820906-50bb5180-ed9a-11e8-9517-ee6e6c776d31.png)

But AMP with next source (using amp-mathml) generates the next figure.
```html
<p><amp-mathml layout="container" data-formula="\[x = {-b \pm \sqrt{b^2-4ac} \over 2a}\]"></amp-mathml></p>
<p>MathJax <amp-mathml layout="container" inline data-formula="\[x = {-b \pm \sqrt{b^2-4ac} \over 2a}\]"></amp-mathml> MathML <amp-mathml layout="container" inline data-formula="\[ \cos(θ+φ) \]"></amp-mathml> AMP</p>
```
![image](https://user-images.githubusercontent.com/6624567/48820921-63358b00-ed9a-11e8-9456-66d4bad8aaed.png)

It's slightly different; the inline mode (non-block mode) formula has smaller fractions.

Since MathJax does not produce `div.MJXc-display` when it's inline mode, so we should consider it.

Suggested changes: [mathml.js#L66-L73](https://github.com/ampproject/amphtml/blob/master/3p/mathml.js#L66-L73)
```javascript
          if (display[0]) {
            display[0].setAttribute('style','margin-top:0;margin-bottom:0');
            context.requestResize(
                rendered./*OK*/offsetWidth,
                rendered./*OK*/offsetHeight
            );
            setStyle(div, 'visibility', 'visible');
          }
```
to
```javascript
          if (display[0]) {
            display[0].setAttribute('style','margin-top:0;margin-bottom:0');
          }
          context.requestResize(
              rendered./*OK*/offsetWidth,
              rendered./*OK*/offsetHeight
          );
          setStyle(div, 'visibility', 'visible');
```

Not tested, just my guess. If it's confirmed for change, I will do some build and test.

## What browsers are affected?

Chrome, but seems to happen in all browsers.

## Which AMP version is affected?

1811202351440

**Comment by KENNYSOFT** [comment]
Wanna mention @adamsilverstein :smile:

**Comment by torch2424** [comment]
Going to triage to @aghassemi , as I don't know what this would fall under 😂 Feel free to re assign 😄

**Comment by aghassemi** [comment]
@KENNYSOFT Thanks for debugging and proposing a solution. It seems good to me. @adamsilverstein thoughts?

@KENNYSOFT if you don't mind, I suggest testing your solution a bit more and sending a PR for the fix. That would be very much appreciated.

Thanks

**Comment by KENNYSOFT** [comment]
@aghassemi Absolutely. I will do some fix with tests, and I think that `inline` attribute could be deleted also. Thanks for confirming. 👍

**Comment by aghassemi** [comment]
Thanks @KENNYSOFT! Regarding deleting `inline`, it would be a backward-incompatible change, so would need to leave it as is.

**Comment by KENNYSOFT** [comment]
@aghassemi Oh, I see. Thanks for comment!

**Comment by KENNYSOFT** [comment]
#19436 Have no idea why ci in PR fails 😭
